Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -712,50 +712,50 @@ WebPageProxy::WebPageProxy(PageClient& pageClient, WebProcessProxy& process, Ref
     , m_navigationState(makeUnique<WebNavigationState>())
     , m_generatePageLoadTimingTimer(RunLoop::main(), this, &WebPageProxy::didEndNetworkRequestsForPageLoadTimingTimerFired)
     , m_legacyMainFrameProcess(process)
-    , m_pageGroup(*m_configuration->pageGroup())
-    , m_preferences(m_configuration->preferences())
-    , m_userContentController(m_configuration->userContentController())
+    , m_pageGroup(*configuration->pageGroup())
+    , m_preferences(configuration->preferences())
+    , m_userContentController(configuration->userContentController())
```

Comment:
this is a use-after-move

Reply:
You're totally right--thank you! Fixed.

Reply:
fine if we need to do this for the static analyzer, but I think the previous code with the move was safe. m_configuration is already a Ref and could be made const.